api/napi/src/interpreter/component_instance.rs

```diff
@@ -36,8 +36,19 @@ impl JsComponentInstance {
     }
 
     #[napi]
-    pub fn run(&self) {
-        self.inner.run().unwrap()
+    pub fn show(&self, env: Env) -> Result<JsUndefined> {
```

Comment:
Why is this needed?  this doesn't seem to be called as the TS does `this.instance.window().show();`  (note the `window()` )

Reply:
Well spotted - thanks!